gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -58,15 +35,30 @@
 import com.linkedin.restli.server.resources.BaseResource;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
-
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
this is not correct. please follow https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/ and use gobblin's codestyle

Reply:
import order restored by code style